modules/modelSetup/StableDiffusionLoRASetup.py

```diff
@@ -45,9 +46,13 @@ def create_parameters_for_optimizer(
             args: TrainArgs,
     ) -> Iterable[Parameter] | list[dict]:
         param_groups = list()
+        batch_size = 1 if args.learning_rate_scaler in [LearningRateScaler.NONE, LearningRateScaler.GRADIENT_ACCUMULATION] else args.batch_size
+        gradient_accumulation_steps = 1 if args.learning_rate_scaler in [LearningRateScaler.NONE, LearningRateScaler.BATCH] else args.gradient_accumulation_steps
```

Comment:
Thanks for keeping this configurable, because it'd be a horrible idea for adaptive algos.